flow-typed/stripe.js

```diff
@@ -0,0 +1,87 @@
+import * as STRIPE from 'constants/stripe';
+
+declare type StripeState = {
```

Comment:
:small_red_triangle: Types should not depend on constants (those are generated at eval time)

One solution is to define the currency as a union of literals, and that use that in the constant file instead. "Flow enums" is another, but I haven't tried that before.

```
declare type CurrencyOption = 'USD' | 'EUR';
```
  - The IDE won't pick it up correctly...
    - <img width="319" alt="image" src="https://user-images.githubusercontent.com/64950861/185527166-1590bf34-1dc5-452e-a91d-3d6e6aeaf4b0.png">
    - <img width="257" alt="image" src="https://user-images.githubusercontent.com/64950861/185527197-99a8aa42-c697-4f94-bbe5-548e5ccb7cd9.png">
  - ... until I removed that import statement
    - <img width="330" alt="image" src="https://user-images.githubusercontent.com/64950861/185527276-b28a0ab5-cfb3-4d66-a9cb-a2a806ca6b40.png">

Reply:
And thanks for defining the types :heart: